dark/views/tournament/team/member/join.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib import messages
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.views.generic.base import View
+
+from dark.models.tournament.team import Team, TeamRole, TeamMember
+
+
+class JoinTeamMemberView(View):
+    def get(self, request, tournament, team):
```

Comment:
Chyba da się to zrobić za pomocą CreateView + ForeignKeysMixin + get_context_data

Reply:
Próbowałem coś takiego zrobić, ale tak naprawdę nie bardzo ogarniam to wszystko. Myślę że niech to zostanie, może jakoś w przyszłości do tego wrócę ;)